ciphey/basemods/Decoders/galactic.py

```diff
@@ -0,0 +1,98 @@
+# community
+# by https://github.com/lukasgabriel
+
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
+
+@registry.register
+class Galactic(Decoder[str, str]):
+    def decode(self, ctext: T) -> Optional[U]:
+        """
+        Takes a string written in the 'Standard Galactic Alphabet' 
+        (aka Minecraft Enchanting Table Symbols) and translates it to ASCII text.
+        """
+
+        result = ""
+        galactic_letters = [
+            "ᔑ",
+            "ʖ",
+            "ᓵ",
+            "↸",
+            "ᒷ",
+            "⎓",
+            "⊣",
+            "⍑",
+            "╎",
+            "⋮",
+            "ꖌ",
+            "ꖎ",
+            "ᒲ",
+            "リ",
+            "𝙹",
+            "!",
+            "ᑑ",
+            "∷",
+            "ᓭ",
+            "ℸ",
+            "⚍",
+            "⍊",
+            "∴",
+            "|",
+            "⨅",
+        ]
+        letters = list("abcdefghijklmnopqrstuvwyz")
+        galactic_dict = {galactic_letters[i]: letters[i] for i in range(25)}
```

Comment:
This is does not need to be computed at run time. Run this locally in Python idle, and then upload to CipheyDIsts :)

Reply:
Sorry but you'll have to tell me how to do that 😆

Reply:
Sure!
1. Load up Python idle (in a terminal, just type Python). 
2. Copy and paste the galactic list, and the letters list into idle as variables.
So you're in idle and:
```
> letters = list("abcdefghijklmnopqrstuvwyz")
>
```

3. Run your variable `{galactic_letters[i]: letters[i] for i in range(25)}`
This will print out the dictionary you're using
4. Fork CipheyDists https://github.com/Ciphey/CipheyDists
5. In this folder, create a file `galactic.json` https://github.com/Ciphey/CipheyDists/tree/master/cipheydists/translate
6. Copy and paste the dictionary you have from Python into here.
7. Copy and paste here to make sure its valid JSON https://jsonlint.com/
8. pull request to CipheyDists
9. Copy and paste how we did it for morse code https://github.com/Ciphey/Ciphey/blob/master/ciphey/basemods/Decoders/morse.py#L98 into your own code
10. I _think_ that might work? If not, Discord :P

Reply:
Thanks, will do!

Reply:
I adapted my init method as follows: ```    def __init__(self, config: Config):
        super().__init__(config)
        self.GALACTIC_DICT = config.get_resource(
            self._params()["dict"], WordList
        )```

and am getting the following traceback:```Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\gabrielu\AppData\Local\pypoetry\Cache\virtualenvs\ciphey-nUZHK-CA-py3.7\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\gabrielu\AppData\Local\pypoetry\Cache\virtualenvs\ciphey-nUZHK-CA-py3.7\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "C:\Users\gabrielu\AppData\Local\pypoetry\Cache\virtualenvs\ciphey-nUZHK-CA-py3.7\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\gabrielu\AppData\Local\pypoetry\Cache\virtualenvs\ciphey-nUZHK-CA-py3.7\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\ciphey.py", line 270, in main
    result = decrypt(config, kwargs["text"])
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\ciphey.py", line 40, in decrypt
    res: Optional[iface.SearchResult] = config.objs["searcher"].search(ctext)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\basemods\Searchers\ausearch.py", line 219, in search
    self.recursive_expand(root)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\basemods\Searchers\ausearch.py", line 200, in recursive_expand
    self.recursive_expand(new_node)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\basemods\Searchers\ausearch.py", line 200, in recursive_expand
    self.recursive_expand(new_node)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\basemods\Searchers\ausearch.py", line 200, in recursive_expand
    self.recursive_expand(new_node)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\basemods\Searchers\ausearch.py", line 190, in recursive_expand
    inst = self._config()(decoder)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\iface\_config.py", line 108, in __call__
    return self.instantiate(t)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\iface\_config.py", line 103, in instantiate
    ret = t(self)
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\basemods\Decoders\galactic.py", line 49, in __init__
    self._params()["dict"], WordList
  File "C:\Users\gabrielu\Documents\GitHub\Ciphey\ciphey\iface\_modules.py", line 84, in _params
    return self._params_obj
AttributeError: 'Galactic' object has no attribute '_params_obj'
```

Reply:
Might be because the pull request on the CipheyDists isn't through yet, but idk 🤔